.github/workflows/reusable-native-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Reusable - Native tests
+
+on:
+  workflow_call:
+    inputs:
+      skip-native-tests:
+        type: boolean
+        required: false
+
+jobs:
+  graalvm-native-tests:
+    runs-on: ubuntu-latest
```

Comment:
I think here you need to add a line like `if: "!inputs.skip-native-tests"` to actually skip running the test when the label was missing

Reply:
`reusable-native-tests.yml` is used in the daily GraalVM native worflow: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/a8f6599d7f51f0a708d1d3e7ada2c2b3bfceea72/.github/workflows/native-tests-daily.yml#L11

The goal of the label is to be able to execute the GraalVM native tests for some PRs. This label is not necessary for the daily check.

Reply:
Oh, the daliy build seems to work fine but I have just seen that the GraalVM native tests are executed in the current PR.

Reply:
The GraalVM native tests are now skiped in the current PR. @laurit could you review again?